### DISCUSSION TITLE
2021.06.09

### DISCUSSION THREAD

**Discussion by malaheaven** [malaheaven/issue-tracker#15] [General]
회의: api 형식 수정

**Comment by Settpark** [comment]
iOS: 두번째 뷰 그리고 BE와 같이 api 회의 했습니다. iOS는 오늘 행복합니다.

**Comment by 17-sss** [comment]
**FE**

오늘 다 같이 API 회의를 했습니다!  
그리고 화면 공유하며 페어프로그래밍을!  
열심히 UI를 짜는데 필요한 theme(팔레트)를 만들었슴다!
FE도 행복합니다 😄╰(*°▽°*)╯😄

**Comment by malaheaven** [comment]
### BE
오늘 API회의를 했습니다. 그리고 mock api 제작해서 드렸어요
화면 공유하며 페어프로그래밍을 진행했어요 
BE도 행복합니닼ㅋㅋㅋㅋㅋㅋㅋㅋ
